### DISCUSSION TITLE
Spring Boot Starter 2.7.0, Spring Boot 3. 3.2 and RestTemplate Exchange

### DISCUSSION THREAD

**Discussion by sraveh** [open-telemetry/opentelemetry-java-instrumentation#12091] [Q&A] [answered]
Hi,

In my project Im trying to integrate Open Telemetry Spring Boot Starter 2.7.0. My project is using spring boot 3.3.2. I have integration test cases that are using RestTemplate  exchange method with a Get Operation. After Including the Open Telemetry Spring Boot Started dependencies, those tests are now falling with the following error:  "java.lang.IllegalArgumentException: method GET must not have a request body."

This is thrown from "io.opentelemetry.instrumentation.spring.web.v3_1.RestTemplateInterceptor.intercep"

**Comment by trask** [comment, marked as answer]
hi @sraveh! can you open an issue for this and include a repro that can be used to troubleshoot the issue?

**Reply by sraveh** [reply]
Thanks for the response, after further tests, I discovered that the issue was with my code reusing the same http body between requests. You can close this issue/ discussion.

Thanks,
Simon
